src/build_platform/local_docker.rs

```diff
@@ -518,52 +518,33 @@ fn check_docker_space_usage_and_clean(
     ))
 }
 
-fn docker_prune_images(envs: Vec<(&str, &str)>) -> Result<Vec<String>, SimpleError> {
-    let docker_image_prune_args = vec!["image", "prune", "-a", "-f"];
-    let docker_builder_prune_args = vec!["builder", "prune", "-a", "-f"];
-
-    let result_docker_image_prune = cmd::utilities::exec_with_envs_and_output(
-        "docker",
-        docker_image_prune_args,
-        envs.clone(),
-        |line| {
-            let line_string = line.unwrap();
-            debug!("{}", line_string.as_str());
-        },
-        |line| {
-            let line_string = line.unwrap();
-            debug!("{}", line_string.as_str());
-        },
-        Duration::minutes(BUILD_DURATION_TIMEOUT_MIN),
-    );
-
-    match result_docker_image_prune {
-        Ok(_) => {}
-        Err(e) => return Err(e),
-    };
-
-    let result_docker_builder_prune = cmd::utilities::exec_with_envs_and_output(
-        "docker",
-        docker_builder_prune_args,
-        envs,
-        |line| {
-            let line_string = line.unwrap();
-            debug!("{}", line_string.as_str());
-        },
-        |line| {
-            let line_string = line.unwrap();
-            debug!("{}", line_string.as_str());
-        },
-        Duration::minutes(BUILD_DURATION_TIMEOUT_MIN),
-    );
-
-    match result_docker_builder_prune {
-        Ok(mut x) => {
-            let mut image_prune_lines = result_docker_image_prune.unwrap();
-            image_prune_lines.append(&mut x);
-            let all_lines = image_prune_lines;
-            Ok(all_lines)
-        }
-        Err(x) => Err(x),
+fn docker_prune_images(envs: Vec<(&str, &str)>) -> Result<(), SimpleError> {
+    let all_prunes_commands = vec![
+        vec!["container", "prune", "-f"],
```

Comment:
Why not ` docker system prune -a -f` ?

Reply:
Because it removes networks which breaks docker then if you don't restart the daemon, which is not possible because of PID 1